Infrastructure-Scripts/Remove-AppServiceIpException.ps1

```diff
@@ -0,0 +1,44 @@
+<#
+    .SYNOPSIS
+    Remove the access restriction rules from an app service
+
+    .DESCRIPTION
+    Remove the access restriction rules from an app service given an IP
+
+    .PARAMETER IpAddress
+    An ip address to associate with the access restriction rule
+
+    .PARAMETER ResourceName
+    The name of the app service
+
+    .EXAMPLE
+    Remove-AppServiceIpException -IpAddress 192.168.0.1 -ResourceName das-prd-sms-as
+#>
+
+[CmdletBinding()]
+Param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNull()]
+    [IPAddress]$IpAddress,
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNull()]
+    [String]$ResourceName
+)
+
+
+$AppServiceResource = Get-AzResource -Name $ResourceName -ResourceType "Microsoft.Web/sites"
+
+if (!$AppServiceResource) {
+    throw "Could not find a resource matching $ResourceName in the subscription"
+}
+
+$AppServiceWhitelist = Get-AzWebAppAccessRestrictionConfig -ResourceGroupName $AppServiceResource.ResourceGroupName -Name $ResourceName | Where-Object { $_.MainSiteAccessRestrictions.IpAddress -eq "$IPAddress/32" }
+
+if (!$AppServiceWhitelist) {
+    Write-Output " -> Could not find whitelisted $IPAddress to remove!"
```

Comment:
Could all the logging statements say what app service is having its access restrictions changed?